### DISCUSSION TITLE
Open Beta Program

### DISCUSSION THREAD

**Discussion by guacamole** [containerish/OpenRegistry#137] [Announcements]
## What's Changed
* Fix: renamed github workflow names by @guacamole in https://github.com/containerish/OpenRegistry/pull/78
* Fix: Use our fork of go-skynet for all Skynet Ops by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/82
* Feat: Support for Pinning to Skynet using single account by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/84
* Fix: logging by adding new logger struct by @guacamole in https://github.com/containerish/OpenRegistry/pull/86
* Postgres implementation by @guacamole in https://github.com/containerish/OpenRegistry/pull/87
* Yaml app config by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/91
* Chore: GitGuradian Secret Scanning by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/92
* Feat: implemented pagination api by @guacamole in https://github.com/containerish/OpenRegistry/pull/93
* Chore: Create LICENSE by @guacamole in https://github.com/containerish/OpenRegistry/pull/94
* Chore(deps): Bump github.com/docker/distribution from 2.7.1+incompatible to 2.8.0+incompatible by @dependabot in https://github.com/containerish/OpenRegistry/pull/96
* Chore: Dependency Cleanup by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/97
* Feat: added api to list user catalog with pagination by @guacamole in https://github.com/containerish/OpenRegistry/pull/98
* Refactor: Using Skynet SDK's Metadata API by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/100
* Chore: Adds cancel-in-progress on consecutive push by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/101
* Chore: OCI Certification Badge by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/105
* Chore: updated Readme MD to include disclaimer by @guacamole in https://github.com/containerish/OpenRegistry/pull/107
* Update issue templates by @guacamole in https://github.com/containerish/OpenRegistry/pull/109
* Feat: basic implementation for search for auto-completion api by @guacamole in https://github.com/containerish/OpenRegistry/pull/112
* Fix: Skynet Portal URL from config file by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/113
* Fix: removed io.ReadAll from entire project by @guacamole in https://github.com/containerish/OpenRegistry/pull/114
* Feat: added console writer for local logging by @guacamole in https://github.com/containerish/OpenRegistry/pull/123
* Add: updated http client for skynet by @guacamole in https://github.com/containerish/OpenRegistry/pull/124
* GitHub auth by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/95
* Renew access token by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/126
* Add: logging for newly merged code by @guacamole in https://github.com/containerish/OpenRegistry/pull/127
* Refactor: removed all occurrences of KV/Badger store by @guacamole in https://github.com/containerish/OpenRegistry/pull/128
* Refactor: removed session struct and auth from types.go by @guacamole in https://github.com/containerish/OpenRegistry/pull/129
* Add: sort images with namespace and last_updated by @guacamole in https://github.com/containerish/OpenRegistry/pull/130
* Feat: email verification by @guacamole in https://github.com/containerish/OpenRegistry/pull/131
* Password management apis by @guacamole in https://github.com/containerish/OpenRegistry/pull/132
* Enhanced Logging by @guacamole in https://github.com/containerish/OpenRegistry/pull/134
* Final changes before release by @jay-dee7 in https://github.com/containerish/OpenRegistry/pull/136

## New Contributors
* @dependabot made their first contribution in https://github.com/containerish/OpenRegistry/pull/96

**Full Changelog**: https://github.com/containerish/OpenRegistry/compare/v0.1.0...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/containerish/OpenRegistry/releases/tag/v0.2.0'>Open Beta Program</a>.</em>

**Comment by AlaniAKT** [comment]
Thanks for sharing this! Fantastic work!
